images/php/tests/main.tf

```diff
@@ -32,5 +32,5 @@ data "oci_exec_test" "composer" {
 data "oci_exec_test" "fpm" {
   count  = var.check-fpm ? 1 : 0
   digest = var.digest
-  script = "docker run --rm $IMAGE_NAME -v"
+  script = "FPMRUN=$( timeout --preserve-status -s TERM 4s docker run --rm $IMAGE_NAME 2>&1 ) && echo $FPMRUN | grep -q 'ready to handle connections'"
```

Comment:
just a nit - it might be easier to maintain if this was moved into a bash script, as then we could run it locally without applying the terraform.  Not blocking the merge, just a thought for the future.

Reply:
going to resolve the comment as it is non blocking